wled00/data/index.js

```diff
@@ -974,9 +974,15 @@ function makeWS() {
 		displayRover(info, s);
 		readState(json.state);
 	};
-	ws.onclose = function(event) {
-    	d.getElementById('connind').style.backgroundColor = "#831";
-  	}
+	ws.onclose = (e)=>{
+		d.getElementById('connind').style.backgroundColor = "#831";
+		ws = null;
+		if (lastinfo.ws > -1) setTimeout(makeWS,500); //retry WS connection
+	}
+	ws.onopen = (e)=>{
+		ws.send("{'v':true}");
```

Comment:
@blazoncek One tip for your fork, this line is actually unneeded :)
WLED WS will always send `si` automatically upon initial connection
https://github.com/Aircoookie/WLED/blob/master/wled00/ws.cpp#L18